apps/play/pages/games/cryptoUnicorns.tsx

```diff
@@ -12,85 +12,857 @@ import {
 import { supportedChains } from "../../../../types/Moonstream";
 import { useERC20, useToast } from "moonstream-components/src/core/hooks";
 import { useMutation, useQuery } from "react-query";
 import { DEFAULT_METATAGS } from "../../src/constants";
-import { MAX_INT } from "moonstream-components/src/core/providers/Web3Provider";
+import {
+  MAX_INT,
+  chainByChainId,
+} from "moonstream-components/src/core/providers/Web3Provider";
+import LootboxCard from "../../../../packages/moonstream-components/src/components/CryptoUnicorns/LootboxCard";
+import { MockTerminus as TerminusFacet } from "../../../../types/contracts/MockTerminus";
+import { hookCommon } from "moonstream-components/src/core/hooks";
 
-const contractsAddr: { [key in supportedChains]: string } = {
+const terminusAbi = require("../../../../abi/MockTerminus.json");
+
+const GameBankAddresses: { [key in supportedChains]: string } = {
   mumbai: "0x762aF8cbE298bbFE568BBB6709f854A01c07333D",
   polygon: "0x94f557dDdb245b11d031F57BA7F2C4f28C4A203e",
   ethereum: "0x0000000000000000000000000000000000000000",
   localhost: "0x0000000000000000000000000000000000000000",
 };
+
+const TerminusAddresses: { [key in supportedChains]: string } = {
+  mumbai: "0x19e812EdB24B68A8F3AbF5e4C82d10AfEf1641Db",
+  polygon: "0x99A558BDBdE247C2B2716f0D4cFb0E246DFB697D",
+  ethereum: "0x0000000000000000000000000000000000000000",
+  localhost: "0x0000000000000000000000000000000000000000",
+};
+
+const UNIMAddresses: { [key in supportedChains]: string } = {
+  mumbai: "0x47d0f0BD94188e3f8c6fF2C0B1Bf7D6D8BED7534",
+  polygon: "0x64060aB139Feaae7f06Ca4E63189D86aDEb51691",
+  ethereum: "0x0000000000000000000000000000000000000000",
+  localhost: "0x0000000000000000000000000000000000000000",
+};
+
+const RBWAddresses: { [key in supportedChains]: string } = {
+  mumbai: "0x4Df452487E6c9d0C3Dc5EB4936244F8572b3F0b6",
+  polygon: "0x431CD3C9AC9Fc73644BF68bF5691f4B83F9E104f",
+  ethereum: "0x0000000000000000000000000000000000000000",
+  localhost: "0x0000000000000000000000000000000000000000",
+};
+
+const MulticallAddresses: { [key in supportedChains]: string } = {
+  mumbai: "0xe9939e7Ea7D7fb619Ac57f648Da7B1D425832631",
+  polygon: "0xc8E51042792d7405184DfCa245F2d27B94D013b6",
+  ethereum: "0x0000000000000000000000000000000000000000",
+  localhost: "0x0000000000000000000000000000000000000000",
+};
+
+const UnicornsAddresses: { [key in supportedChains]: string } = {
+  mumbai: "0x39858b1A4e48CfFB1019F0A15ff54899213B3f8b",
+  polygon: "0xdC0479CC5BbA033B3e7De9F178607150B3AbCe1f",
+  ethereum: "0x0000000000000000000000000000000000000000",
+  localhost: "0x0000000000000000000000000000000000000000",
+};
+
+const LandsAddresses: { [key in supportedChains]: string } = {
+  mumbai: "0x230E4e85d4549343A460F5dE0a7035130F62d74C",
+  polygon: "0xA2a13cE1824F3916fC84C65e559391fc6674e6e8",
+  ethereum: "0x0000000000000000000000000000000000000000",
```

Comment:
We can get rid of `ethereum` and `localhost` keys. Not so useful and add a lot of noise to the code.